server/internal/infrastructure/memory/container.go

```diff
@@ -12,3 +12,14 @@ func New() *repo.Container {
 		Lock:        NewLock(),
 	}
 }
+
+func InitRepos(c *repo.Container) *repo.Container {
+	if c == nil {
+		c = &repo.Container{}
+	}
+	c.Workspace = NewWorkspace()
+	c.User = NewUser()
+	c.Transaction = NewTransaction()
+	c.Lock = NewLock()
+	return c
+}
```

Comment:
```suggestion
func New() *repo.Container {
	c := &repo.Container{}
	c.Workspace = NewWorkspace()
	c.User = NewUser()
	c.Transaction = NewTransaction()
	c.Lock = NewLock()
	return c
}
```

Follow https://github.com/reearth/reearth-backend/pull/155/files#diff-e5b56ca6e4472add4e461f6f230a29980e0c43ae5797b6d024833447eeb45358